sgl-kernel/csrc/cpu/torch_extension_cpu.cpp

```diff
@@ -209,8 +212,9 @@ TORCH_LIBRARY_FRAGMENT(sgl_kernel, m) {
 
   // decode
   m.def(
-      "decode_attention_cpu(Tensor query, Tensor output, Tensor k_cache, Tensor v_cahce, Tensor attn_logits, Tensor "
-      "req_to_token, Tensor req_pool_indices, Tensor seq_lens, float sm_scale, float logit_cap) -> ()");
+      "decode_attention_cpu(Tensor query, Tensor k_cache, Tensor v_cache, Tensor output, Tensor key, Tensor "
```

Comment:
This change is needed to fix a building error on main branch.
This can be removed when https://github.com/sgl-project/sglang/pull/6404 lands.